rbd-target-api.py

```diff
@@ -503,7 +503,7 @@ def get_config():
     """
 
     if request.method == 'GET':
-        config.refresh()
+        config.get()
```

Comment:
Nit: why not just re-use the existing `get_config` method?

Reply:
good point, I will update

Reply:
Done